c7n/resources/iam.py

```diff
@@ -1157,6 +1157,7 @@ class UserDelete(BaseAction):
     ORDERED_OPTIONS = OrderedDict([
         ('console-access', 'delete_console_access'),
         ('access-keys', 'delete_access_keys'),
+        ('attached-user-policies', 'delete_attached_user_policies'),
         ('user-policies', 'delete_user_policies'),
```

Comment:
if we're going to specify attached vs not, can you change `user-policices` to `inline-user-policies`?, same thing with the function name as well `delete_user_policies` -> `delete_inline_user_policies`

Reply:
you are right, and it's the official AWS lingo, but AWS use user-policies in the api and cli. Should we still change it?

Reply:
Yeah, from an interface standpoint we're already changing the aws convention to provide a clearer interface when defining policies, i.e. `console-access` is not `login-profile`. I personally find `delete_inline_user_policies` and `delete_attached_user_policies` to be much clearer wrt the intention of the action itself.

Reply:
@thisisshi I changed it as to your suggestion